### PR TITLE
RET-1232

### PR DESCRIPTION
**Pull request by olyode**
### JIRA link
https://tools.hmcts.net/jira/browse/RET-1232
### Change description
Merged notice end and notice pay for contents
**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: src/test/unit/controller/NoticeEndController.test.ts [hunk 1]
```diff
@@ -17,4 +17,12 @@ describe('Notice end Controller', () => {
     noticeEndController.get(request, response);
     expect(response.render).toHaveBeenCalledWith(TranslationKeys.NOTICE_END, expect.anything());
   });
```
Comment (marcjudge) [inline src/test/unit/controller/NoticeEndController.test.ts:19]: Please include a test for a successful post

File: src/main/definitions/case.ts [hunk 2]
```diff
@@ -32,8 +32,12 @@ export interface Case {
   presentEmployer?: YesOrNo;
   typeOfClaim?: TypesOfClaim[];
   pastEmployer?: YesOrNo;
+  noticeEnd?: string;
   noticePeriod?: YesOrNo;
   noticePeriodLength?: string;
+  noticePeriodUnit2?: WeeksOrMonths;
```
Comment (marcjudge) [inline src/main/definitions/case.ts:38]: I don't like the names of these variables with 2 at the end. They aren't very descriptive. Please make sure you aren't duplicating values that already exist like noticePeriodLength. It looks like that value could probably be refactored to be WeeksOrMonths and shared between the 'working' and 'notice period' flows.

File: src/test/unit/controller/NoticeEndController.test.ts (hunk 1 above)
Comment (olyode) [reply src/test/unit/controller/NoticeEndController.test.ts:19]: Added

File: src/main/resources/locales/en/translation/notice-end.json [hunk 3]
```diff
@@ -1,3 +1,10 @@
 {
-  "h1": "Employment details - notice end (to do)"
+  "h1": "When does your notice period end?",
+  "p1": "Tell us when you’ll finish with them.",
+  "p2": "For example, 22 04 2014.",
+  "errors": {
+    "noticeEnds": {
+      "required": "Enter your employment start date. For example, 22 04 2014"
```
Comment (marcjudge) [inline src/main/resources/locales/en/translation/notice-end.json:7]: Looks like the wrong error message

File: src/test/unit/controller/NoticeEndController.test.ts (hunk 1 above)
Comment (marcjudge) [reply src/test/unit/controller/NoticeEndController.test.ts:19]: Not seeing a post test added here

File: src/test/routes/notice-pay.test.ts [hunk 4]
```diff
@@ -17,7 +17,7 @@ describe(`on POST ${PageUrls.NOTICE_PAY}`, () => {
       .post(PageUrls.NOTICE_END)
```
Comment (marcjudge) [inline src/test/routes/notice-pay.test.ts:17]: You're describing a notice_pay test bust then testing a post with notice_end. A successful post with notice_pay should redirect to the average weekly hours page

File: src/test/routes/notice-end.test.ts [hunk 5]
```diff
@@ -12,7 +12,7 @@ describe(`GET ${PageUrls.NOTICE_END}`, () => {
 });
 
 describe(`on POST ${PageUrls.NOTICE_END}`, () => {
-  test('should navigate to the notice pay page when save and continue button is clicked', async () => {
+  test('should navigate to the notice end page when save and continue button is clicked', async () => {
```
Comment (marcjudge) [inline src/test/routes/notice-end.test.ts:15]: Test text was correct previously, 'should navigate to the notice **pay** page when save and continue button is clicked'

**Review by marcjudge — CHANGES_REQUESTED** [review]

**Review by marcjudge — APPROVED** [review]

**Review by robin-ieong — APPROVED** [review]

**Pull request merged**